homework-6/script-1.js

```diff
@@ -0,0 +1,78 @@
+//Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+function getMinNumber(num_1, num_2, num_3, num_4) {
```

Comment:
There is no validation for `null`

<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191333270-d326a47c-cc1c-4859-9b3d-deb8bbd62457.png">